topology/probes/ovsdb.go

```diff
@@ -46,6 +46,7 @@ type OvsdbProbe struct {
 	Graph        *graph.Graph
 	Root         *graph.Node
 	OvsMon       *ovsdb.OvsMonitor
+	OvsOfProbe   *OvsOfProbe
```

Comment:
Sorry forgot to commit my answer on using a graph listener. Using OvsMonitor would be the worst solution. I want to know that OvsdbProbe.OnOvsBridgeProbe is finished. And the best way to do that is calling my callback at the end of the other one. Using the OvsMonitor, I would not even be sure I am called after the ovsdb probe.